extensions/keycloak-admin-client-reactive/runtime/src/main/java/io/quarkus/keycloak/admin/client/reactive/runtime/ResteasyReactiveClientProvider.java

```diff
@@ -24,15 +24,22 @@
 import io.quarkus.arc.InstanceHandle;
 import io.quarkus.jackson.ObjectMapperCustomizer;
 import io.quarkus.rest.client.reactive.jackson.runtime.serialisers.ClientJacksonMessageBodyWriter;
+import io.quarkus.runtime.TlsConfig;
 
 public class ResteasyReactiveClientProvider implements ResteasyClientProvider {
 
     private static final List<String> HANDLED_MEDIA_TYPES = List.of(MediaType.APPLICATION_JSON);
     private static final int PROVIDER_PRIORITY = Priorities.USER + 100; // ensures that it will be used first
 
+    private final TlsConfig tlsConfig;
```

Comment:
Out of curiosity, do you have a reason to store `TlsConfig` instead of just that boolean?

Reply:
good point. let me change that.